usaspending_api/search/v2/views/spending_by_geography.py

```diff
@@ -397,6 +402,59 @@ def district_results(self, state_lookup: str, geo_queryset: QuerySet) -> List[di
 
         return results
 
+    def country_results(self, loc_lookup: str) -> List[dict]:
+        """Find subaward results for countries
+
+        Args:
+            loc_lookup (String): Name of the field on the SubawardSearch model to use to find subawards.
+
+        Returns:
+            List[dict]: List of subaward results by country
+        """
+        country_queryset = self.queryset.values(loc_lookup)
+
+        # If specific countries were provided, only get the subawards for those countries
+        if self.geo_layer_filters:
+            country_queryset = country_queryset.filter(**{f"{loc_lookup}__in": self.geo_layer_filters})
+
+            ref_countries = RefCountryCode.objects.filter(country_code__in=self.geo_layer_filters).values(
+                "country_code", "country_name"
+            )
+            ref_countries = {country["country_code"]: country["country_name"] for country in ref_countries}
+        # If no specific countries were provided, then get all subawards grouped by country
+        else:
+            ref_countries = RefCountryCode.objects.all().values("country_code", "country_name")
+            ref_countries = {country["country_code"]: country["country_name"] for country in ref_countries}
```

Comment:
Not a big deal, but if line 427 matches 423 exactly, it should be brought out of the `if/else` block

Reply:
I didn't notice that I duplicated that line. I'll add a commit that pulls it out of the `if/else` block.

Reply:
Cleaned up the duplicate line in commit d11dd90